framework/src/main/java/org/checkerframework/framework/stub/StubTypes.java

```diff
@@ -427,6 +444,8 @@ private void prepJdk11FromFile(URL resourceURL) {
                     walk.filter(p -> Files.isRegularFile(p) && p.toString().endsWith(".java"))
                             .collect(Collectors.toList());
             for (Path path : paths) {
+                // TODO: revert.
```

Comment:
Highlighting that there are two `TODO: revert` comments.

Reply:
Thanks for catching this! I had merged in a different branch that had changes that should not be merged.  I've reverted them now.